README.md

```diff
@@ -1,17 +1,33 @@
+A simple Go based Yum metadata parser with fetching capabilities.
+
+Currently this project only parses package information, more may be added in the future. 
```

Comment:
```suggestion
Currently this project only parses package information, more might be added in the future. 
```
"may" means permission, try "might" or "can" as appropriate to the context.
In this case you could drop ", more might be added in the future" as it does not add any value.